tests/bricks/test_conv.py

```diff
@@ -51,3 +54,65 @@ def test_max_pooling():
     pool.input_dim = (x_size, y_size)
     pool.get_dim('output') == (num_channels, x_size / pool_size + 1,
                                y_size / pool_size + 1)
+
+
+class ConvNN(Sequence, Initializable, Feedforward):
```

Comment:
I guess we'll want to move this to `bricks/conv.py`, because this is useful beyond having a test only. I think @harmdevries89 was working in something similar though (a wrapper for multiple convolutional layers), so not sure which implementation has more features?